src/main/java/net/pms/dlna/DLNAMediaAudio.java

```diff
@@ -430,7 +430,7 @@ public String getAudioCodec() {
 		if (is3GA()) {
 			return "3GA";
 		} else if (isAACLC()) {
-			return "AAC";
+			return "AAC-LC";
 		} else if (isAC3()) {
```

Comment:
Yes, or ```FormatConfiguration.AAC_LC```. The point is that if all references in the code is to this, instead of hardcoding the string, it's much easier to refactor later without breaking stuff. When using strings directly, one have to actually find all the places in the code where it's used, and that can be very difficult and easily leads to regressions.